src/getHumanAge.test.js

```diff
@@ -7,4 +7,28 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should work with zeros', () => {
+    const result = getHumanAge(0, 0);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test('should work correct with young animals', () => {
+    const result = getHumanAge(10, 10);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test('should work correct with older animals', () => {
+    const result = getHumanAge(15, 15);
+
+    expect(result).toEqual([1, 1]);
+  });
+
+  test('should work correct with old animals', () => {
+    const result = getHumanAge(64, 74);
+
+    expect(result).toEqual([12, 12]);
```

Comment:
The expected result for old animals should be verified. Make sure that the function's logic for calculating human age equivalents is correct and the expected output is accurate.